src/Acuminator/Acuminator.Analyzers/StaticAnalysis/DacExtensionDefaultAttribute/DacExtensionDefaultAttributeAnalyzer.cs

```diff
@@ -140,9 +146,8 @@ public static async Task<Location> GetAttributeLocationAsync(AttributeData attri
 		{
 			SyntaxNode attributeSyntaxNode = null;
```

Comment:
@saloponov the declaration of the variable should be written with the following assignment:
```
var attributeSyntaxNode = await attribute.ApplicationSyntaxReference.GetSyntaxAsync(cancellationToken).ConfigureAwait(false);
```